umm-spec-lib/src/cmr/umm_spec/umm_spec_core.clj

```diff
@@ -152,4 +152,4 @@
      mt/dif (dif-util/parse-access-constraints metadata true)
      mt/dif10 (dif-util/parse-access-constraints metadata true)
      mt/iso19115 (iso19115-2-to-umm/parse-access-constraints metadata true)
-     mt/iso-smap nil)))
+     nil)))
```

Comment:
The code in this namespace looks like it's missing conversions for parsing collection temporal and collection access-value when the format is UMM-JSON as well as parsing a collection access-value when the collection is in ISO SMAP format. It seems like we are missing some tests and might have some bugs.

Reply:
@chris-durbin: Can we go over this after standup tomorrow?

Reply:
I have no idea if this is even suppose to throw an exception for other formats.  It could be that its assumed to never be called in that case and there should be an error.  I'm thinking this could affect production for collections that are native umm-json.  I submitted a PR that had the smallest footprint, but I can try and add a test case for this.

Reply:
If this is breaking MMT I'm good with adding your fix in for now since I can't see how it would make things any worse. We should open a new ticket to deal with the formats that are missing conversions.